src/libraries/System.Net.Http/tests/StressTests/HttpStress/HttpStress.csproj

```diff
@@ -4,8 +4,19 @@
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <Nullable>enable</Nullable>
     <EnablePreviewFeatures>True</EnablePreviewFeatures>
+    <NoWarn>CA2252</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <!-- Shared production code -->
+  <ItemGroup>
+    <Compile Include="..\..\..\..\Common\src\System\Net\Logging\NetEventSource.Common.cs" Link="Common\System\Net\Logging\NetEventSource.Common.cs" />
+    <Compile Include="..\..\..\..\System.Net.Quic\src\System\Net\Quic\Internal\MsQuicApi.cs" Link="ProductionCode\System\Net\Quic\Internal\MsQuicApi.cs" />
+    <Compile Include="..\..\..\..\System.Net.Quic\src\System\Net\Quic\Internal\MsQuicSafeHandle.cs" Link="ProductionCode\System\Net\Quic\Internal\MsQuicSafeHandle.cs" />
+    <Compile Include="..\..\..\..\System.Net.Quic\src\System\Net\Quic\Interop\*.cs" Link="ProductionCode\System\Net\Quic\Interop\*.cs" />
+    <Compile Include="..\..\..\..\System.Net.Quic\src\System\Net\Quic\NetEventSource.Quic.cs" Link="ProductionCode\System\Net\Quic\NetEventSource.Quic.cs" />
```

Comment:
A refactor of these internals may break stress builds without us noticing, but I guess there is no good way to deal with this. We should remember that `HttpStress` took a dependency.

Reply:
Also: these files are missing from the docker context on Windows, which is why the Windows build failed. Copying them would make this solution even more fragile. Can we implement worker delay tuning hack with reflection instead and share that implementation between `System.Net.Quic.Functional.Tests` and the stress tests? That would also remove the need of including anything but `QuicDefaults.cs` in `System.Net.Quic.Functional.Tests.csproj`.

Reply:
Ah no, we would need #75347 for to invoke function pointers with reflection safely. I guess, the best would be to [copy](https://github.com/dotnet/runtime/blob/d27142772e8de8c47c97a7661f5bf03f1a2ff703/eng/docker/build-docker-sdk.ps1#L52-L53) & include `msquic_generated.cs`, then do a combination of reflection + code that only depends on the generated types.

I can take over if you want.

Reply:
Reflection was one of the options we discussed with @CarnaViire when implementing this in functional tests. I was able to workaround it in functional tests with source code inclusion, which is normal for corresponding tests. However, here it proves problematic.
I could re-work this to create an internal function on `MsQuicApi` which we would invoke via reflection from both functional tests as well as stress.